### ISSUE TITLE
Adding Border

### ISSUE THREAD

**Issue by systems-rebooter** [uttesh/ngletteravatar#5]
Hey,

I'm trying to add border, but it shows only top and left
Can you assist, please?

```
}).css({
                                        'background-color': color,
                                        'width': width+'px',
                                        'height': height+'px',
                                        'border' : 'solid 5px',
                                        'border-bottom-style': 'dotted'
 });
```

![screen shot 2015-06-23 at 19 57 16](https://cloud.githubusercontent.com/assets/8888190/8311901/1d3e599a-19e2-11e5-8ab3-b487e6ffad9c.png)

**Comment by systems-rebooter** [comment]
Even `convert -border 2x2 -bordercolor "#000000" 1.png 1-border.png` did the trick for me. I'm just curious how to achieve this on code side

![download 3](https://cloud.githubusercontent.com/assets/8888190/8315003/f4ec948e-19f5-11e5-9433-07556b07b844.png)

**Comment by uttesh** [comment]
Hey, 

its look very nice with border, yes i made changes in the directive added avatar border feature attribute, its released in v2.0.1 available in bower and demo page updated with changes.

![avatar_border1](https://cloud.githubusercontent.com/assets/2218611/8315091/a9942fea-1a0b-11e5-967d-6995e117aef3.png)
![avatar_border2](https://cloud.githubusercontent.com/assets/2218611/8315094/ad279674-1a0b-11e5-9635-dc7f5acd7ec3.png)
![border_attribute](https://cloud.githubusercontent.com/assets/2218611/8315097/b638285a-1a0b-11e5-8fcd-7d08fdcc9834.png)

**Comment by systems-rebooter** [comment]
Wow, you are fast! Thanks a lot. Right now I have at least 2 choices :) Huge +1
